packages/create-plugin/templates/github/ci/.github/workflows/release.yml

```diff
@@ -8,11 +8,12 @@ on:
     tags:
       - 'v*' # Run workflow on version tags, e.g. v1.0.0.
 
-permissions:
-  contents: write
+permissions: read-all
```

Comment:
As we only have a single job in this workflow, what's the reason for this change at this point?